metriq/models/moderationReport.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+from tea_client.models import TeaClientModel
+
+from metriq.models.page import Page
+
+
+class ModerationReport(TeaClientModel):
+    """ModerationReport object.
+
+    Attributes:
+        id (str): ModerationReport ID.
+        name (str): ModerationReport name.
+        fullName (str): Full name of ModerationReport.
+        description (str): Task description.
+        resolvedAt (str): when ModerationReport was resolved
+    """
+
+    class Config:
+        fields = {'id': '_id'}
+
+    id: str
+    userId: Optional[str]
+    name: Optional[str]
+    fullName: Optional[str]
+    description: Optional[str]
+    resolvedAt: Optional[str]
+
+
+
+class ModerationReports(Page):
```

Comment:
There is no `GET` route for moderation reports, so this isn't needed. Please remove.